device/.cargo/config.toml

```diff
@@ -1,5 +1,5 @@
 [target.riscv32imc-unknown-none-elf]
-runner = "espflash flash --baud 921600 --bootloader bootloader.bin"
+runner = "espflash flash --baud 921600 --bootloader bootloader.debug.bin"
```

Comment:
what's going on here?

Reply:
Ah, this was me looking at 2nd stage bootloader logs before trying to enable flash encryption. Pushing new commit my bad